grpc/ra-wrappers.go

```diff
@@ -30,22 +30,8 @@ func (rac RegistrationAuthorityClientWrapper) NewRegistration(ctx context.Contex
 	return rac.inner.NewRegistration(ctx, request)
 }
 
-func (rac RegistrationAuthorityClientWrapper) NewAuthorization(ctx context.Context, authz core.Authorization, regID int64) (core.Authorization, error) {
-	req, err := AuthzToPB(authz)
-	if err != nil {
-		return core.Authorization{}, err
-	}
-
-	response, err := rac.inner.NewAuthorization(ctx, &rapb.NewAuthorizationRequest{Authz: req, RegID: regID})
-	if err != nil {
-		return core.Authorization{}, err
-	}
-
-	if response == nil || !authorizationValid(response) {
-		return core.Authorization{}, errIncompleteResponse
-	}
-
-	return PBToAuthz(response)
+func (rac RegistrationAuthorityClientWrapper) NewAuthorization(ctx context.Context, request *rapb.NewAuthorizationRequest) (*corepb.Authorization, error) {
+	return rac.inner.NewAuthorization(ctx, &rapb.NewAuthorizationRequest{Authz: request.Authz, RegID: request.RegID})
```

Comment:
Think this can just be
```suggestion
	return rac.inner.NewAuthorization(ctx, request)
```

Reply:
Good catch!